### PR TITLE
Create ControllerStates class to make it easier to pass to Command

### PR DESCRIPTION
**Pull request by tran-tien-dat**

### CODE REVIEW

File: src/test/java/cs2103/v15_1j/jimjim/command/UnmarkCommandTest.java [hunk 1]
```diff
@@ -1,98 +0,0 @@
-package cs2103.v15_1j.jimjim.command;
```
Comment (shaunthium) [inline src/test/java/cs2103/v15_1j/jimjim/command/UnmarkCommandTest.java]: Why did you delete this file?

Comment (tran-tien-dat) [inline src/test/java/cs2103/v15_1j/jimjim/command/UnmarkCommandTest.java]: it's a duplicate. Probably because of a merge conflict
